templates/navbar.mustache

```diff
@@ -0,0 +1,121 @@
+{{!
```

Comment:
As you fork a new template to boost_union, please add the current upstream version to templates/navbar.mustache.upstream as well to ease future diffs with the upstream version